src/utils.jl

```diff
@@ -301,7 +301,7 @@ end
     """ ->
     function gplot(sm::SimResult)
         N = length(sm.colnames)
-        figure()
+        Gaston.figure()
```

Comment:
I don't know, just started getting some Tk errors with Winston recently.
PyPlot also has some build errors, but I have not investigated very much.

On Wed, Feb 18, 2015 at 10:09 AM, Tom Short <notifications@github.com>
wrote:

> In src/utils.jl
> <https://github.com/tshort/Sims.jl/pull/57#discussion_r24924464>:
>
> > @@ -301,7 +301,7 @@ end
> >      """ ->
> >      function gplot(sm::SimResult)
> >          N = length(sm.colnames)
> > -        figure()
> > +        Gaston.figure()
>
> Interesting. What's broken about Winston? Also, I've been liking PyPlot
> lately.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tshort/Sims.jl/pull/57/files#r24924464>.
>